doctr/local.py

```diff
@@ -248,3 +248,20 @@ def check_repo_exists(deploy_repo, service='github', *, auth=None, headers=None)
     r.raise_for_status()
 
     return r.json().get('private', False)
+
+def in_git_root():
+    """
+    Check if current working directory is root dir of git repo.
+    Return ``False`` if it isn't.
+    """
+    repo = subprocess.run(['git', 'rev-parse', '--show-toplevel'],
```

Comment:
Well it doesn't have to be at the root of the repo, unless I'm mistaken, as long as the paths to the key and the other arguments are given to it, relative to the current directory. It does need to be in the repo, as it calls git. 

So I actually think it should only error when it is not in the git repo (it will error anyway, eventually, because git will complain as soon as doctr calls it), and warn, but not error when it isn't, just so it is easier for people to debug if they accidentally are cd'd to the wrong directory and are trying to figure out why doctr failed.